structuredefinitions/England-MedicationRequest.xml

```diff
@@ -172,6 +172,10 @@
         <profile value="https://fhir.nhs.uk/StructureDefinition/Extension-England-EPSDispensingInformation" />
       </type>
       <isModifier value="false" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="https://fhir.nhs.uk/ValueSet/England-DMPrescriptionEndorsement" />
```

Comment:
Where does this change come from? Can't find the ticket about it

Reply:
Good spot. Should have mapped ValueSet-England-DMMedicationDispenseType to Extension.extension:dispenseStatus.
Item 4 of the Feedback email.